### PR TITLE
Feature/payment ledger upload to settlement

### PR DESCRIPTION
**Pull request by yuseok-kim-edushare**
This pull request introduces several important changes to improve error handling, add new functionality, and update navigation links. The most significant changes include adding error handling to the `getProfileImage` function, implementing the `appendLedger` function, and updating the navigation link for the home page.

Error handling improvements:

* [`apps/client/src/action/profile/getProfilePic.ts`](diffhunk://#diff-a80d75d90d19faeff6aa2c0bf262bdfa43f96e158710add4620f43d411ca54c1R13): Added a try-catch block to handle errors when fetching the profile image, logging errors to the console, and returning a default response in case of failure. [[1]](diffhunk://#diff-a80d75d90d19faeff6aa2c0bf262bdfa43f96e158710add4620f43d411ca54c1R13) [[2]](diffhunk://#diff-a80d75d90d19faeff6aa2c0bf262bdfa43f96e158710add4620f43d411ca54c1R33-R40)

New functionality:

* [`apps/client/src/action/settlement/ledgerAppendAction.ts`](diffhunk://#diff-a45e727c576e871ec629084bc45dddbcb4d31a7a1a8ae6313db26b60fccf60f7R1-R52): Implemented the `appendLedger` function to transform payload items, fetch seller information, and append the ledger with the transformed data.

Integration with payment success page:

* [`apps/client/src/components/payment/SuccessPage.tsx`](diffhunk://#diff-02b411cfe8124f715f229817fe46842de6d47566d9939f2fdbf1915db8b9e2e8L5-R6): Imported the `appendLedger` function and called it with the purchase list data upon successful payment. [[1]](diffhunk://#diff-02b411cfe8124f715f229817fe46842de6d47566d9939f2fdbf1915db8b9e2e8L5-R6) [[2]](diffhunk://#diff-02b411cfe8124f715f229817fe46842de6d47566d9939f2fdbf1915db8b9e2e8R93-R95)

Navigation updates:

* [`apps/client/src/lib/navigation.ts`](diffhunk://#diff-cddb49d789cd4d6c2ac2165fa66a67d6a1ea0e6ab9cba5fd23f75d4460dfd834L49-R49): Updated the home page navigation link from `/home` to `/`.

### CODE REVIEW

File: apps/client/src/components/payment/SuccessPage.tsx [hunk 1]
```diff
@@ -89,6 +90,9 @@ export function SuccessPage() {
 					purchaseList: JSON.parse(json.metadata.paymentList),
 				}
 
+				const purchaseList = JSON.parse(json.metadata.paymentList) as PaymentItemType[]
```
[the remark this answers is not in the document]
Comment (yuseok-kim-edushare) [reply apps/client/src/components/payment/SuccessPage.tsx:93]: Well, hell of lint
thus, we need to do this

File: apps/client/src/action/settlement/ledgerAppendAction.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        const sellerData = await sellerRes.json();
+        
+        if (!isValidResponse<SellerResponse>(sellerData)) {
+            throw new Error(`Invalid seller response format for product ${item.productUUID}`);
+        }
+
+        return {
+            sellerUUID: sellerData.result.sellerUuid,
+            productName: item.productName,
+            price: parseInt(item.productPrice),
```
[the remark this answers is not in the document]
Comment (yuseok-kim-edushare) [reply apps/client/src/action/settlement/ledgerAppendAction.ts:36]: that will ensured our project logic
thus this re parsing unnecessary

**Review by sing-u8 — APPROVED** [review]
확인완료

**Review by GwendolyNM — APPROVED** [review]
확인했습니다

**Pull request merged**